module3/lessons/consuming_an_api_ruby.md

```diff
@@ -0,0 +1,248 @@
+---
+layout: page
+title: Consuming an API with Ruby
+tags: rails, ruby, apis
+---
+
+# Consuming an API with Ruby
```

Comment:
This went really smoothly - I was able to get everything set up and working.  My only note is that the branch name on the turingschool repo is `consumping-api-start` (with an extra 'p').

Reply:
hahah woof. Will fix that!